### PR TITLE
add counter button ,toggle logo button and form with button that triggers input field password type

### PR DESCRIPTION
**Pull request by DenverBrian**
# REQUIREMENTS

- [x] done creating 2 buttons
- [x] Button 1(Counter) - done with functionality
- [x] Button 2(Toggle Vite) -done with functionality

- [x] Listen to events
- [x] Form that has password input and submit button 
- [x] Button outside form to trigger password input type
- [x] Show logo when the number in the counter is even

### CODE REVIEW

**Review by directormac — CHANGES_REQUESTED** [review]
Please finish everything before we merge your pr.

**Comment by DenverBrian** [comment]
> Please finish everything before we merge your pr.

noted

**Review by directormac — CHANGES_REQUESTED** [review]
Add functionaly within the newly created elements

File: src/denver/index.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ if (Page) {
     counterButton.textContent = `clicked ${clicks} times`;
   });
 
+  //FIXME: change approach
```
Comment (directormac) [inline src/denver/index.ts:37]: Go ahead seems redundant to put an initial value of false, when the element itself is not appended yet.

Comment (directormac) [reply src/denver/index.ts:37]: @DenverBrian

Comment (DenverBrian) [reply src/denver/index.ts:37]: noted

**Review by directormac — CHANGES_REQUESTED** [review]
Do it now

File: src/denver/index.ts (hunk 1 above)
Comment (directormac) [inline src/denver/index.ts:37]: Implement the changes ASAP

**Comment by boosquack** [comment]
### Complete task requirements

- Done creating the 2 Buttons (Counter, Toggle Vite) with the functionalities needed to operate.
- Done adding the functionalities with event listeners.
- Completed  form part which includes password input and a submit button.
- Done creating another button to trigger the password type input.

**Pull request merged**